rest/src/main/java/discord4j/rest/route/Routes.java

```diff
@@ -1281,6 +1281,24 @@ public abstract class Routes {
      */
     public static final Route GUILD_SCHEDULED_EVENT_USERS_GET = Route.get("/guilds/{guild.id}/scheduled-events/{event.id}/users");
 
+    ///////////////////////////////////////////
+    ////////////////// Polls //////////////////
+    ///////////////////////////////////////////
+
+    /**
+     * Returns a list of users that voted for the given `answer_id` in the poll for the given message represented by
+     * its `message.id` and `channel.id`.
+     * @see <a href="https://discord.com/developers/docs/resources/poll#get-answer-voters">https://discord.com/developers/docs/resources/poll#get-answer-voters</a>
+     */
+    public static final Route GET_POLL_ANSWER_VOTERS = Route.get("/channels/{channel.id}/polls/{message.id}/answers/{answer_id}");
```

Comment:
```suggestion
    public static final Route POLL_ANSWER_VOTERS_GET = Route.get("/channels/{channel.id}/polls/{message.id}/answers/{answer_id}");
```
for keep the format of the fields name?